assignment-1/src/app/chat-bubble/chat-bubble.component.css

```diff
@@ -0,0 +1,30 @@
+.chat-bubble{
+    font-size: 12px;
+    word-break: break-all;
+    hyphens: auto;
+}
+
+.time{
+    color: #c6c4c5;
+    text-align: end;
+    padding-top: 5px;
+    font-size: 10px;
```

Comment:
nit: try to use variables for fonts and colours from SCSS

Reply:
Variables was used in css